src/components/input/Input.tsx

```diff
@@ -0,0 +1,48 @@
+import { Eye, EyeOff } from "lucide-react";
+import { InputHTMLAttributes, useState } from "react";
+
+interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
+  label: string;
+  error?: string;
+}
+
+function Input({ label, error, type = "text", className, ...rest }: InputProps) {
+  const [showPassword, setShowPassword] = useState(false);
+  const isPasswordType = type === "password";
+
+  return (
+    <div className="flex flex-col">
+      <label className="text-16-m text-gray-950 mb-1">{label}</label>
+      <div className="relative">
+        <input
+          type={isPasswordType ? (showPassword ? "text" : "password") : type}
```

Comment:
1. `label`도 props가 존재할 때만 렌더링하면 좋을 것 같아요 👍
2. `label` 요소의 `htmlFor`속성과 `input`의 `id`속성을 동일하게 전달해서 연결해줘야 해요.

```suggestion
function Input({ id, label, error, type = "text", className, ...rest }: InputProps) {
  const [showPassword, setShowPassword] = useState(false);
  const isPasswordType = type === "password";

  return (
    <div className="flex flex-col">
      {label && <label htmlFor={id} className="text-16-m text-gray-950 mb-1">{label}</label>}
      <div className="relative">
        <input
          id={id}
          type={isPasswordType ? (showPassword ? "text" : "password") : type}
```